3506/0/ruby_koans/triangle.rb

```diff
@@ -0,0 +1,31 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+# :reek:TooManyStatements
+def triangle(side_a, side_b, side_c)
+  side_a, side_b, side_c = [side_a, side_b, side_c].sort
+  raise TriangleError if side_a <= 0
+  raise TriangleError if side_a + side_b <= side_c
+
+  array = [side_a, side_b, side_c].uniq
+  return :equilateral if array.size == 1
+  return :isosceles if array.size == 2
```

Comment:
By the way, you can make it without return) Just think about merging this 4 rows together